distributed/src/test/scala/asobu/distributed/End2EndSpec.scala

```diff
@@ -56,8 +58,11 @@ object End2EndSpec {
 
   object DistributedSystem {
 
+    val interpreter = new Interpreter {
+      override def interpret[T](cred: CustomRequestExtractorDefinition[T]): RequestExtractor[T] = ???
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of ??? operator](https://www.codacy.com/app/iHeartRadio/asobu/file/2681756966/issues/source?bid=3037259&fileBranchId=3335316#l62)